core/app/models/spree/tax_rate.rb

```diff
@@ -16,51 +6,31 @@ class TaxRate < Spree::Base
     include Spree::AdjustmentSource
 
     belongs_to :zone, class_name: "Spree::Zone", inverse_of: :tax_rates
-    belongs_to :tax_category, class_name: "Spree::TaxCategory", inverse_of: :tax_rates
+    belongs_to :tax_category,
+      class_name: "Spree::TaxCategory",
+      inverse_of: :tax_rates
 
     validates :amount, presence: true, numericality: true
     validates :tax_category_id, presence: true
-    validates_with DefaultTaxZoneValidator
 
-    scope :by_zone, ->(zone) { where(zone_id: zone) }
-
-    def self.potential_rates_for_zone(zone)
-      select("spree_tax_rates.*, spree_zones.default_tax").
-        joins(:zone).
-        merge(Spree::Zone.potential_matching_zones(zone)).
-        order("spree_zones.default_tax DESC")
-    end
+    scope :by_zone, -> (zone) { where(zone_id: zone.id) }
+    scope :potential_rates_for_zone,
+      -> (zone) do
+        where(zone_id: Spree::Zone.potential_matching_zones(zone).pluck(:id))
+      end
+    scope :for_default_zone,
+      -> { potential_rates_for_zone(Spree::Zone.default_tax) }
```

Comment:
This one as well. Is there a way in rails to associate over to the zone model and call `zone.class.default_tax`?